src/main/java/mate/academy/model/Book.java

```diff
@@ -1,4 +1,29 @@
 package mate.academy.model;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Read about using lombok's data on jpa entities
replace it with Getter Setter annotaitons